source/future.tex

```diff
@@ -98,20 +98,58 @@
 prefix \tcode{++} and \tcode{\dcr} expressions\iref{expr.pre.incr}
 of \tcode{volatile}-qualified arithmetic and pointer types are deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile int velociraptor;
+++velociraptor;                 // deprecated: is this a single memory operation, two, or a six-foot turkey?
+\end{codeblock}
+\end{example}
+
+
 \pnum
 Certain assignments
 where the left operand is a \tcode{volatile}-qualified non-class type
 are deprecated; see~\ref{expr.ass}.
 
+\begin{example}
+\begin{codeblock}
+int neck, tail;
+volatile int brachiosaur;
+tail = brachiosaur = neck;      // deprecated: does this touch brachiosaur once or twice?
```

Comment:
```suggestion
tail = brachiosaur = neck;      // deprecated: does this access brachiosaur once or twice?
```

Reply:
It's not clear from this example which of the two assignment expressions in this statement is deprecated.

Reply:
Could we have a `volatile_lhs += rhs` example in here as well?